examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1696,8 +1801,9 @@ def compute_text_embeddings(prompt, text_encoders, tokenizers):
                     variant=args.variant,
                     torch_dtype=weight_dtype,
                 )
-
                 pipeline_args = {"prompt": args.validation_prompt}
+                if args.do_edm_style_training:
+                    pipeline_args.update({"guidance_scale": 5.0})
```

Comment:
Not in favour of this. value of `guidance_scale` isn't really related to `EDM`, here we should use the pipeline default or accept this as arg.